game-headed/src/test/java/org/triplea/game/client/ui/javafx/screens/MainMenuPaneTest.java

```diff
@@ -33,21 +33,24 @@ void testGetNode() {
   void testInitialize() throws Exception {
     final NavigationPane mock = mock(NavigationPane.class);
     final StackPane mock2 = mock(StackPane.class);
-    FieldSetter.setField(null, Application.class.getDeclaredField("userAgentStylesheet"), "");
+    final Field userAgentStyleSheetField =
+        Application.class.getDeclaredField("userAgentStylesheet");
+    userAgentStyleSheetField.setAccessible(true);
+    userAgentStyleSheetField.set(null, "");
```

Comment:
What's the 'set' of null needed for? Could this be extracted to a method to make it self-documenting, or alternatively worth explaining in a quick comment.